### ISSUE TITLE
Latest changes not published to NPM

### ISSUE THREAD

**Issue by lztetreault-dev** [apexcharts/ng-apexcharts#351]
The changes I made a few weeks ago that are the last merge in this repository have not been released as an update to NPM.

**Comment by julianpoemp** [comment]
Does your PR fix the issue that a chart is not rendered after first initialization? I've defined the options in the constructor and referenced their attributes to the input fields like that:

````html
<apx-chart [series]="options.series!" [chart]="options.chart" [dataLabels]="options.dataLabels" [stroke]="options.stroke" [title]="options.title" [grid]="options.grid" [xaxis]="options.xaxis"></apx-chart>
````

On Initialization no chart is visible, if I resize the window it's rendered

**Comment by ltetreault-va** [comment]
I have not checked that, but I assume that would not be fixed by my PR since it only made changes to stop Angular from complaining when 'strictTemplates' is enabled.

**Comment by junedchhipa** [comment]
released `ng-apexcharts@1.12.0`
